aries-site/src/pages/components/layer.mdx

```diff
@@ -52,6 +52,28 @@ For example, implementation of this "Close" button for a Layer would be:
 />
 ```
 
+## Types 
+
+Depending on the content displayed, Layers may be thought of as either [informational](components/layer#informational) or [actionable](components/layer#actionable). This distinction is important as it affects whether the page content adjacent to the Layer should be interactive, as well as impacting how a user may close the Layer.
+
+### Informational
+Used purely to inform users of status or state. These are text-only with no required or optional actions. These may be [dialog modals](/components/layer#center-modal), [toast notifications](/templates/toast-notifications), etc.
+
+Informational Layers contain the following elements and interactions:
+- A sole ["X" icon](https://icons.grommet.io/?s=formclose) for the user to close out of the Layer. Additional elements, such as a close button, cause further clutter and confusion for the user.
+- For user generated layers such as directly opened dialog model, the Layer can be closed by clicking outside of it's content. 
```

Comment:
```suggestion
- For dialog modals that open as a result of a user clicking an element on the page, the Layer can be closed by clicking outside of its content. 
```
I'm not fully clear on what this means -- suggesting a wording change based on how i'm interpreting it.